stl_reader.h

```diff
@@ -280,11 +291,6 @@ class StlMesh {
 
     return res;
   }
-
-  bool read_file (const std::string& filename)
-  {
-    return read_file (filename.c_str());
-  }
```

Comment:
Sorry, I didn't see this last time. But it seems that you removed this method? I think it is in the public interface, so some users may rely on it. Unless there's a good reason to remove it, I would thus rather keep it. And if we have an `std::string` overload, I think it would even make sense to add a second overload with `const std::wstring&`, as you did in the constructor...